geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/operations/GetAllRequestOperationHandler.java

```diff
@@ -63,13 +67,14 @@
       }
       return Success.of(RegionAPI.GetAllResponse.newBuilder().addAllEntries(entries).build());
     } catch (UnsupportedEncodingTypeException ex) {
+      logger.debug("Received GetAll request with unsupported encoding: " + ex);
       return Failure.of(ProtobufResponseUtilities.makeErrorResponse(
           ProtocolErrorCode.VALUE_ENCODING_ERROR.codeValue, "Encoding not supported."));
     } catch (CodecNotRegisteredForTypeException ex) {
+      logger.debug("Got codec error when decoding GetAll request: " + ex);
```

Comment:
I would like to see that we handle exceptions as exceptions and log them via error or warning messages. Logging these failures on a debug level, would potentially hide/mask issues that could be root-caused by seeing that an exception was thrown and handled.

Reply:
In the revision I just pushed this particular exception is a error, but I made the one above that's for a unsupported encoding info level. My reasoning is that the exception about unsupported encoding is caused by user/client error whereas the codec not registered exception is caused by us advertising we support the codec, but then we can't deliver. I strongly agree that the later should be a highly visible. Someone sending a request with a codec we don't support in an ideal world shouldn't even be a exception though, since we should expect and faulty requests and they therefore should go through the regular control flow.